Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -697,7 +699,8 @@ public bool RemoveRenamedComponent(VBComponent component, string oldComponentNam
 
             if (success)
             {
-                OnStateChanged();
+                OnStateChanged(ParserState.ResolvedDeclarations);   // trigger test explorer and code explorer updates
+                OnStateChanged(ParserState.Ready);   // trigger find all references &c. updates
```

Comment:
three. there's a smell here. who's *not* firing `ResolvedDeclarations` when they should?